spec/actions_md_spec.rb

```diff
@@ -3,7 +3,7 @@
     @actions_md = File.read("docs/Actions.md")
     local_actions = ["rubocop"]
     discontinued = ["update_project_code_signing"]
-    others = ["default_platform", "gradle"]
+    others = ["default_platform", "gradle", "adb", "adb_devices"]
```

Comment:
Please remove `adb` and `adb_devices` from the exceptions list and add documentation for them to [Actions.md](https://github.com/fastlane/fastlane/blob/master/docs/Actions.md). (Gradle should also be documented but haven't gotten a chance to do that! 🙊)